ts/client/src/utils/rpc.ts

```diff
@@ -37,6 +39,29 @@ export async function sendTransaction(
 
   const payer = (provider as AnchorProvider).wallet;
 
+  //
+  // setComputeUnitLimit, hard code to a higher minimum, this is needed so that we dont fail simple UI interactions
+  //
+  // https://github.com/solana-labs/solana-web3.js/blob/master/packages/library-legacy/src/programs/compute-budget.ts#L202
+  const computeUnitLimitIxFound = ixs.some(
+    (ix) =>
+      ix.programId.equals(
+        new PublicKey('ComputeBudget111111111111111111111111111111'),
```

Comment:
sadly these constants are not importable

Reply:
very sad